index.js

```diff
@@ -36,5 +36,13 @@ function getMessage (name) {
     `Have no fear, ${name} is here!`
   ];
 
-  return messages[Math.floor(Math.random() * messages.length)];
+  const signOff_messages = [
+    `${name} says: Hasta la vista...baby :gun: :sunglasses:`,
+    `${name} is peacing out :v:`,
+    `It's time for ${name} to go night night :wave:`
+  ];
+
+  return (isSignOn) 
+    ? signOn_messages[Math.floor(Math.random() * signOn_messages.length)]
```

Comment:
Let's abstract this into a named method rather than having it copy/pasted like this.

```js
function getRandomMessage (messages) {
  return messages[Math.floor(Math.random() * messages.length)]
}
```